dbt/models/eia923/core_eia923__monthly_generation_fuel_nuclear/schema.yml

```diff
@@ -15,7 +15,7 @@ sources:
           - name: fuel_type_code_pudl
             data_tests:
               - dbt_expectations.expect_column_values_to_be_in_set:
-                  value_set: ["nuclear"]
+                  value_set: ["nuclear", "oil"]
```

Comment:
This seems weird, but it's because there's diesel backup power at nuclear plants that gets used rarely.